docs/getting-started/raster/raster_step-by-step.md

```diff
@@ -67,6 +94,31 @@ body (XML):
 </csw:GetRecords>
 ```
 
+3. Assuming you enquire all raster products, you can use the following query
```

Comment:
I think the numbering is wrong, step one i aquiring the productId, there are a few ways to do it:
1. query for mc:productType=OrthophotoBest
2. query for all raster products

The numbering alongside the steps are confusing.
Let's find a better way to re-organize the queries and steps

Reply:
fixed